lib/api/core/statistics.js

```diff
@@ -255,7 +255,11 @@ class StatisticsController {
 function writeStats () {
   this.lastFrame = Date.now();
 
+<<<<<<< HEAD
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Parse error at line 258 column 2:

248:     this.timer = setTimeout(() => writeStats.call(this), this.interval);
249:   }
250: }
251: 
252: /**
253:  * @this StatisticsController
254:  */
255: function writeStats () {
256:   this.lastFrame = Date.now();
257: 
258: <<<<<<< HEAD
      ^
259:   this.kuzzle.cacheEngine.internal.setex(
260: =======
261:   this.kuzzle.services.internalCache.setex(
262: >>>>>>> 2-dev
263:     this.cacheKeyPrefix + this.lastFrame,
264:     this.ttl,
265:     JSON.stringify(this.currentStats));
266: 
267:   this.currentStats.completedRequests = {};
268:   this.currentStats.failedRequests = {}; [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AParsingError)